.github/workflows/build.yml

```diff
@@ -79,5 +79,5 @@ jobs:
     - name: make baresip macOS
       if: ${{ runner.os == 'macOS' }}
       run: |
-        make V=1 CCACHE= info test modules
-        make clean; make CCACHE= STATIC=yes
+        make V=1 CCACHE= SYSROOT_LOCAL=/usr/local info info2 test modules
```

Comment:
is `info2` only for debugging used?

Reply:
Yes. Like info. The info target is defined in re.mk. And the variables LIBREM_PATH, ... are not included. Should I remove it or rename it?